decaf377-fmd/src/clue_key.rs

```diff
@@ -14,15 +14,15 @@ use crate::{hash, hkd, Clue, Error, MAX_PRECISION};
 /// situations where clue key might or might not actually be used.  This saves
 /// computation; at the point that a clue key will be used to create a [`Clue`],
 /// it can be expanded to an [`ExpandedClueKey`].
+/// TODO: use Zeroize?
```

Comment:
I don't think it's important to use `Zeroize` for `ClueKey`, because the clue key is public, so there aren't secrets to leak.

To be more precise, the clue key might not be completely public, in the sense that someone might not want to give out their address, and in the sense that it's not included in any transactions, but it doesn't have cryptographic secrets, and this isn't any different level of protection than any other data involved in creating a transaction.

Reply:
Ok, I wasn't sure how toxic the `ClueKey` was. I know how it's used in the spec, and I know we feed it to an HKDF so I thought I'd check. Thanks that makes sense!